### PR TITLE
Rspec

### PR DESCRIPTION
**Pull request by mricanho**
In this project, we added specs for all public methods we created in the Tic Tac Toe project.

### CODE REVIEW

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status Changes Required :recycle: 
Hello @mricanho @vichuge. Nice to see you both here and congratulations for coming till here. :muscle: However, before moving on there are some additions that you need to do.

- [x] At your game spec please apply some negative cases.
commonsense
- [x] For example for your print_winner method, you can assign a winner, and then you can print out this winner instead of nil.
- [x] For the win test method, testing whether the winner is equal to nil is not a real test. For this test please apply a game first by your hands and then expect it to return the condition that you give. And of course, again, implement a negative test case.

Good job on creating a solid layout for the tests. :+1: Please implement these suggested changes and re-submit for a review.
If you have a question about the review, feel free to reach me via Slack. :100: 
_Eyüp Sercan UYGUR_

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @mricanho and @vichuge  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this milestone can be approved.

## REQUIRED

* [x] Make your specs cover positive and negative scenarios (screenshot from Pathwright)
Example: there is a spec for correct usage of the method [e.g. it "creates object if all required attributes are passed"]
and also there is spec for the wrong usage of a method [e.g. it "does not create the object if required attributes are missing"].

## Make the required changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by ghost — APPROVED** [review]
# Status: Approved 🎉

**Congratulations team 👏, and good luck with your next project.**

[Optional] You can add, in your read me instruction manual, the command line for running the RSpec tests.

**Pull request merged**
